### PR TITLE
Enable CI for Jenkins Operator

### PR DESCRIPTION
**Pull request by waveywaves**
Create OWNERS file 
Adding openshift-ci build and {pre,post}submits files

### CODE REVIEW

**Comment by waveywaves** [comment]
cc @sbose78 @akram @bparees

File: ci-operator/config/redhat-developer/jenkins-operator/OWNERS [hunk 1]
```diff
@@ -0,0 +1,18 @@
+approvers:
+- akram
+- waveywaves
+- sbose78
+- otaviof
+- gabemontero
+- bparees
```
Comment (bparees) [inline ci-operator/config/redhat-developer/jenkins-operator/OWNERS]: i don't need to be here.  i'm already in the toplevel config dir OWNERS file.

File: ci-operator/config/redhat-developer/jenkins-operator/redhat-developer-jenkins-operator-master.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  from: base
+  inputs:
+    src:
+      paths:
+        - destination_dir: .
+          source_path: /go/src/github.com/redhat-developer/jenkins-operator/.
+  to: jenkins-operator
+#promotion:
+#  name: "4.2"
+#  namespace: ocp
```
Comment (bparees) [inline ci-operator/config/redhat-developer/jenkins-operator/redhat-developer-jenkins-operator-master.yaml]: with no tests and no promotion i'm not sure what this job is going to do?

File: ci-operator/jobs/redhat-developer/jenkins-operator/OWNERS [hunk 3]
```diff
@@ -0,0 +1,18 @@
+approvers:
+- akram
+- waveywaves
+- sbose78
+- otaviof
+- gabemontero
+- bparees
```
Comment (bparees) [inline ci-operator/jobs/redhat-developer/jenkins-operator/OWNERS]: i don't need to be here, i'm in the toplevle jobs owners file

File: ci-operator/config/redhat-developer/jenkins-operator/redhat-developer-jenkins-operator-master.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      memory: 4Gi
+    requests:
+      cpu: 200m
+      memory: 200Mi
+tag_specification:
+  name: "4.3"
+  namespace: ocp
+#tests:
+#- as: unit
+#  commands: make test
```
Comment (sbose78) [inline ci-operator/config/redhat-developer/jenkins-operator/redhat-developer-jenkins-operator-master.yaml]: Could you add an empty `make test` to your repo and them un-comment his? The idea is to keep test steps here as high-level as possible and then iterate on those make targets in the jenkins repo.

Comment (sbose78) [reply ci-operator/config/redhat-developer/jenkins-operator/redhat-developer-jenkins-operator-master.yaml]: ( could be a separate PR too )

Comment (waveywaves) [reply ci-operator/config/redhat-developer/jenkins-operator/redhat-developer-jenkins-operator-master.yaml]: I just enabled the tests, was waiting for the tests to pass before I added any.

File: ci-operator/jobs/redhat-developer/jenkins-operator/OWNERS (hunk 3 above)
Comment (waveywaves) [reply ci-operator/jobs/redhat-developer/jenkins-operator/OWNERS]: okay, I will remove you from this list then

File: ci-operator/config/redhat-developer/jenkins-operator/redhat-developer-jenkins-operator-master.yaml (hunk 2 above)
Comment (waveywaves) [reply ci-operator/config/redhat-developer/jenkins-operator/redhat-developer-jenkins-operator-master.yaml]: just added unit tests

File: ci-operator/config/redhat-developer/jenkins-operator/OWNERS (hunk 1 above)
Comment (waveywaves) [reply ci-operator/config/redhat-developer/jenkins-operator/OWNERS]: Done

**Pull request merged**